python/mp/effects/bin_demo.py

```diff
@@ -0,0 +1,36 @@
+import copy
+from mp import color as _color
+from mp.effects import effect, bin, looper, sparkle
+
+class BinDemo(effect.Effect):
+    """
+    A demonstration of a composite effect.
```

Comment:
This shows a way to put three effects together into one logical effect by using a bin effect. It also shows more generally how to instantiate sub-effects and invoke next, and that the containing effect can change attributes (twist knobs) on the sub effects.